src/pages/Company/components/CompanyInfo.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import GlobalStyle from '../../../styles/GlobalStyle';
+import theme from '../../../styles/theme';
+import Card from './Card';
+
+function CompanyInfo({ companyData }) {
+  const { jobListings, companyDesc, companyPhoto } = companyData;
+  let joinedDesc = companyDesc.join(' ');
+  const [toggle, setToggle] = useState(false);
+
+  return (
+    <>
+      <GlobalStyle />
+      <ThemeProvider theme={theme}>
+        <CompanyInfos>
+          <InfoSection>
+            <Heading3>채용 중인 포지션</Heading3>
+            <CardWrap>
+              {jobListings &&
+                jobListings.map(data => <Card key={data.id} data={data} />)}
+            </CardWrap>
+          </InfoSection>
+          <InfoSection>
+            <Heading3>회사 소개</Heading3>
+            <PictureWrap>
+              {companyPhoto.map((data, idx) => (
+                <SeparatePictureContainer key={idx}>
+                  <img alt="company photos" src={data} />
+                </SeparatePictureContainer>
+              ))}
+            </PictureWrap>
+            <CompanyIntro height={toggle}>
+              <div>{joinedDesc}</div>
+            </CompanyIntro>
+            <ShowMore
+              onClick={() => {
+                setToggle(!toggle);
+              }}
+            >
+              {toggle ? '접기' : '더보기'}{' '}
```

Comment:
```suggestion
              {toggle ? '접기' : '더보기'}
```
formatting 과정에서 불필요하게 들어가는 부분들 꼼꼼하게 잘 확인해주세요!